roles/microshift_generate_iso/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- include_tasks: "{{ microshift_generate_iso_action }}.yml" # noqa name[missing]
+  when: microshift_generate_iso_action in microshift_generate_iso_action_allowed
```

Comment:
you don't need to make this dynamic, just include install.yml

Reply:
For now yes, but I want to add the cleanup later. I can remove it for now if needed